static/js/giphy.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
+
+const renderGrid = require("@giphy/js-components").renderGrid;
+const GiphyFetch = require("@giphy/js-fetch-api").GiphyFetch;
+const throttle = require("throttle-debounce").throttle;
+
+const gf = new GiphyFetch(page_params.giphy_api_key);
+let search_term = "trending";
+
+exports.renderGrid = function (targetEl) {
+    const render = () =>
+        renderGrid(
+            {
+                width: 300,
+                fetchGifs: (offset) =>
+                    gf.search(search_term, {
+                        offset,
+                        limit: 25,
+                        random_id: page_params.user_id,
```

Comment:
Can you add a comment explaining the choice of `random_id`?  I'm not sure if the goal is for it to be a deterministic function of the Zulip user across sessions, or if it's fine for it to just be a random value that we just generate at the top of this function and changes on every reload.

The latter would be a lot better for privacy.

Reply:
I removed the random_id option since I don't think it makes sense for us to personalize the GIFs for each user as there isn't much to personalize.